Expecto.Tests/Tests.fs

```diff
@@ -701,7 +701,7 @@ let stress =
       testList "single deadlock" [
         testCaseAsync "deadlock" <| async {
           while true do
-            do! Async.Sleep 100
+            do! Async.Sleep 10
```

Comment:
How is this a test for a deadlock? Isn't rather a test for a non-terminating process?